SeQR/app/src/main/java/com/example/seqr/MainActivity.java

```diff
@@ -8,48 +8,70 @@
 import androidx.fragment.app.Fragment;
 import androidx.fragment.app.FragmentManager;
 import androidx.fragment.app.FragmentTransaction;
-
-
+import android.app.Dialog;
+import android.graphics.Bitmap;
 import android.os.Bundle;
+import android.view.Window;
+import android.widget.ImageView;
+import com.google.zxing.BarcodeFormat;
+import com.google.zxing.MultiFormatWriter;
+import com.google.zxing.WriterException;
+import com.google.zxing.common.BitMatrix;
+import com.journeyapps.barcodescanner.BarcodeEncoder;
 import android.util.Log;
 import android.view.Gravity;
 import android.view.MenuItem;
 import android.view.View;
 import android.widget.Button;
 import android.widget.EditText;
 import android.widget.TextView;
-
 import com.example.seqr.models.ID;
 import com.google.android.material.bottomnavigation.BottomNavigationView;
 import com.google.android.material.floatingactionbutton.ExtendedFloatingActionButton;
-
 import com.example.seqr.controllers.ProfileController;
 import com.example.seqr.models.Profile;
 
 public class MainActivity extends AppCompatActivity {
-
-
+    QRCodeGenerator test;
+    //Bitmap map;
+    //Button gen;
+    //ImageView dis;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        
+        //gen = findViewById(R.id.tester);
+        //dis = findViewById(R.id.testdisplay);
+      
+        /*gen.setOnClickListener(new View.OnClickListener() {
```

Comment:
Commented out the generator code for QR, since the UI is changed from Main branch